pkg/agent/multicluster/mc_route_controller.go

```diff
@@ -63,15 +83,16 @@ type MCDefaultRouteController struct {
 	ciImportListerSynced cache.InformerSynced
 	queue                workqueue.RateLimitingInterface
 	// installedCIImports is for saving ClusterInfos which have been processed
-	// in MCDefaultRouteController. Need to use mutex to protect 'installedCIImports' if
-	// we change the number of 'defaultWorkers'.
```

Comment:
Why remove this comment? I think we are still using a single worker and no mutex, right?

Reply:
We removed `defaultWorkers`, but we can keep and update the comments that we need synchronization if we change to multiple workers in future.